build/pom.xml

```diff
@@ -0,0 +1,7 @@
+<project>
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>com.ibm.cohort</groupId>
+  <artifactId>build</artifactId>
+  <version>0.0.1-SNAPSHOT</version>
```

Comment:
I considered the same question and decided not to use parent because the [PMD docs for multimodule Maven projects ](https://maven.apache.org/plugins/maven-pmd-plugin/examples/multi-module-config.html )explicitly did not reference the parent POM. I am imagining a chicken&egg problem where the reactor needs the build dependency for the plugin declaration and it isn't available yet. I didn't actually push on testing it with parent though. Could loop back if folks think it is important.

In terms of 0.0.1-SNAPSHOT vs. ${revision} I don't have any arguments. I can make that change.

Reply:
I'm fine with not referencing the parent pom if the official docs do that.

I guess the version doesn't _really_ matter since the jar is not really useful outside of the build process.  Having it line up with the rest of the projects would look better than endlessly pushing `0.0.1-SNAPSHOT` build jars till the end of time.